src/Test/L0/Listener/Configuration/LinuxServiceControlManagerL0.cs

```diff
@@ -106,7 +106,7 @@ public void VerifyLinuxServiceConfigurationFileContent()
 [Service]
 ExecStart={0}/runsvc.sh
 User={1}
-Environment=NODE_ENV=production
+Environment=
```

Comment:
Please provide guideline, I am happy to do either way.  The only reason I didn't remove the line completely is to show we still have the capability to set "Environment" in the template file.  However with the .Env file we have now, I am not sure if this is the right place to set any env variables.